packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -116,11 +110,7 @@ export const createDefaultCallingHandlers: CreateDefaultCallingHandlers = memoiz
       if (incomingCall) {
         await incomingCall.reject();
       }
-    },
-    /* @conditional-compile-remove(breakout-rooms) */
-    onHangUp: breakoutRoomOriginCall
-      ? async () => breakoutRoomOriginCall.hangUp().then(() => commonCallingHandlers.onHangUp())
-      : commonCallingHandlers.onHangUp
+    }
```

Comment:
Changes here are due the fact that we don't need to hang up main meeting anymore because we are no longer staying on hold in the main meeting after the Calling SDK update.